### DISCUSSION TITLE
Tried so many times to install but still keep running into this issue.

### DISCUSSION THREAD

**Discussion by mujtabaa-adam** [Security-Onion-Solutions/securityonion#13139] [Unsupported Versions]
Hey ya'll. I'm trying to install SecOnion on VMware but the system keeps on "freezing" on this:

![Screenshot 2024-06-04 201047](https://github.com/Security-Onion-Solutions/securityonion/assets/70298285/744dff88-4202-488a-ba0a-8f38ecb04e60)

What do I do??

**Comment by err0r20** [comment]
mushy,

The system may not be "freezing", as you say it is. It could very well still be rsync'ing stuff over from the ISO. I would recommend switching to another tty during the install and maybe check the running processes by using either top or ps. Hope this helps.

**Reply by mujtabaa-adam** [reply]
Thanks for the response. What do you mean by tty, though?

**Comment by dougburks** [comment]
@mushy2005 First, make sure that you verified the ISO image as shown at:
https://securityonion.com/download

Once that's verified, then make sure you're following the hardware requirements at:
https://docs.securityonion.net/en/2.4/hardware.html

Also make sure that you're not oversubscribing the actual hardware on your host.

Here is some other doucmentation that may help you:
https://docs.securityonion.net/en/2.4/vmware.html
https://docs.securityonion.net/en/2.4/trouble-booting.html

**Reply by mujtabaa-adam** [reply]
I've tried looking over all of those resources, nothing seems be able to solve this issue.

**Reply by dougburks** [reply]
@mushy2005 You'll need to provide more information in order for us to help you:

What version are you trying to install?

What are the specs of your VM? (CPU, RAM, storage, etc.)

What are the specs of your host hardware? (CPU, RAM, storage, etc.)

What kind of disk in your host? NVME, SSD, or rotational media?

Are you running any other VMs or apps on the host that could be competing for resources?

**Reply by mujtabaa-adam** [reply]
I managed to get past the installation. But I have another issue now. I keep on getting a "connection timed out" whenever I try to visit the actual interface using the IP address I previously configured. I'm using the latest version of Ubuntu to access the Security Onion's web interface. I've tested pinging both of them with each other and there seems to be a response. I just can't access the actual interface though. For the record, I'm using the latest version of Security Onion.

**Reply by dougburks** [reply]
Have you tried allowing the IP address of your browser to connect through the host-based firewall as shown here?
https://docs.securityonion.net/en/2.4/firewall.html#configuring-host-firewall

**Comment by khoivm67** [comment]
That post-installation step took me up to around 5 hours to finish. How long have you been waiting?
